docker/Dockerfile.test

```diff
@@ -3,7 +3,7 @@ RUN apk add -U --no-cache ca-certificates
 
 WORKDIR /root
 COPY ./bin/starsd /usr/bin/starsd
-COPY ./docker/test-entry-point.sh ./entry-point.sh
```

Comment:
I removed `docker/test-entry-point.sh` because it had the same contents as `docker/entry-point.sh`.